src/button-set.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, {PropTypes as t} from 'react';
+
+// Types
+type ButtonKindType =
+  | 'danger'
+  | 'default'
+  | 'info'
+  | 'primary'
+  | 'secondary'
+  | 'success';
+
+type ButtonType = {
+  text: string,
+  kind: ButtonKindType,
+  onClick: () => any
```

Comment:
Does this allow a function that has parameters? I should know that. Too lazy to look it up. ;-)